src/legacy/core_plugins/data/public/expressions/expression_renderer.tsx

```diff
@@ -19,47 +19,60 @@
 
 import { useRef, useEffect } from 'react';
 import React from 'react';
-import { Ast } from '@kbn/interpreter/common';
 
-import { ExpressionRunnerOptions, ExpressionRunner } from './expression_runner';
-import { Result } from './expressions_service';
+import { ExpressionAST, IExpressionLoaderParams, IInterpreterResult } from './lib/_types';
+import { IExpressionLoader, ExpressionLoader } from './lib/loader';
 
 // Accept all options of the runner as props except for the
 // dom element which is provided by the component itself
-export type ExpressionRendererProps = Pick<
-  ExpressionRunnerOptions,
-  Exclude<keyof ExpressionRunnerOptions, 'element'>
-> & {
-  expression: string | Ast;
+export interface ExpressionRendererProps extends IExpressionLoaderParams {
+  className: 'string';
```

Comment:
@ppisljar The only allowed className here is the literal string "string"- the quotes should be removed